docs/topics/3.3-announcement.md

```diff
@@ -0,0 +1,52 @@
+# Django REST framework 3.3
+
+The 3.3 release marks the final work in the Kickstarter funded series. We'd like to offer a final resounding **thank you** to all our wonderful sponsors and supporters.
+
+The amount of work that has been achieved as a direct result of the funding is immense. We've added a huge amounts of new functionality, resolved nearly 2,000 tickets, and redesigned & refined large parts of the project.
+
+In order to continue 
+
+---
+
+## Release notes
+
+Significant new functionality in the 3.3 release includes:
+
+* Filters presented as HTML controls in the browsable API.
+* A [forms API][forms-api], allowing serializers to be rendered as HTML forms.
+* Django 1.9 support.
+* A [`JSONField` serializer field][jsonfield], corresponding to Django 1.9's Postgres `JSONField` model field.
+* Browsable API support [via AJAX][ajax-form], rather than server side request overloading.
+
+![Filter Controls](../img/filter-controls.png)
+
+*Example of the new filter controls*
+
+## Supported versions
+
+This release drops support for Django 1.5 and 1.6. Django 1.7, 1.8 or 1.9 are now required.
+
+This brings our supported versions into line with Django's [currently supported versions][django-supported-versions]
+
+## Deprecations
```

Comment:
Shall we merge #3478 and mark those features with ?
I'll open a PR for a deprecation page as well.